commit 16a8c21077ea1b6414868726972887fd35e2a82b
Author: Dan McDonald <danmcd@joyent.com>
Date:   2019-07-11T09:47:13-04:00 (3 months ago)
    
    OS-7885 manifest is missing /etc/ipf/smartos_version
    Reviewed by: John Levon <john.levon@joyent.com>
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Approved by: Mike Zeller <mike.zeller@joyent.com>

diff --git a/overlay/generic/manifest b/overlay/generic/manifest
index c273c0d9..2d517eac 100644
--- a/overlay/generic/manifest
+++ b/overlay/generic/manifest
@@ -25,6 +25,8 @@ d etc/inet 0755 root sys
 f etc/inet/ntp.conf 0644 root sys
 f etc/inet/services 0644 root sys
 f etc/inittab 0644 root sys
+d etc/ipf 0755 root sys
+f etc/ipf/smartos_version 0444 root sys
 f etc/issue 0644 root sys
 f etc/logadm.conf 0644 root sys
 f etc/minor_perm 0644 root sys
